File: 0682-baseball-game/0682-baseball-game.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var calPoints = function(operations) {
    const ans = [];
    for (op of operations) {
        if (op === '+') {
            ans.push(ans[ans.length - 1] + ans[ans.length - 2])
        } else if (op === 'D') {
            ans.push(ans[ans.length - 1] * 2)
        } else if (op === 'C') {
            ans.pop()
        } else {
            ans.push(+op)
        }
    }
    return ans.length ? ans.reduce((a,c) => a + c) : 0
};

File: 0682-baseball-game/0682-baseball-game.py
class Solution:
    def calPoints(self, operations: List[str]) -> int:
        ans = []
        for i in range(len(operations)):
            if operations[i] == '+':
                ans.append(ans[len(ans) - 1] + ans[len(ans) - 2])
            elif operations[i] == 'D':
                ans.append(ans[len(ans) - 1] * 2)
            elif operations[i] == 'C':
                ans.pop()
            else:
                ans.append(int(operations[i]))
        count = 0
        for i in range(len(ans)):
            count += ans[i]
        return count
        